### PR TITLE
🍸 Add Kingdom of Loathing presence

### PR DESCRIPTION
**Pull request by gausie**

### CODE REVIEW

File: websites/K/Kingdom of Loathing/dist/metadata.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "color": "#0000ff",
     "tags": ["rpg", "asym", "kol"],
     "category": "games",
+    "url": "kingdomofloathing.com",
```
Comment (ririxidev) [inline websites/K/Kingdom of Loathing/dist/metadata.json:19]: ~~you always need url no matter what~~

File: websites/K/Kingdom of Loathing/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        "name": "gausie",
+        "id": "263438077865754644"
+    },
+    "contributors": [],
+    "service": "Kingdom of Loathing",
+    "altnames": ["KoL", "KoLmafia"],
+    "description": {
+        "en": "Presence for the Kingdom of Loathing"
+    },
+    "version": "0.0.1",
```
Comment (Bas950) [inline websites/K/Kingdom of Loathing/dist/metadata.json:13]: version 1.0.0

File: websites/K/Kingdom of Loathing/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "logo": "https://s3.amazonaws.com/images.kingdomofloathing.com/otherimages/leftswordguy.gif",
+    "thumbnail": "https://s3.amazonaws.com/images.kingdomofloathing.com/otherimages/leftswordguy.gif",
```
Comment (Bas950) [inline websites/K/Kingdom of Loathing/dist/metadata.json:15]: upload to imgur

File: websites/K/Kingdom of Loathing/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+        "name": "gausie",
+        "id": "263438077865754644"
+    },
+    "contributors": [],
```
Comment (Bas950) [inline websites/K/Kingdom of Loathing/dist/metadata.json:7]: remove, contributors is optional

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
remove the zip etc then do new pull request with proofs of working presence

**Pull request closed without merging**